src/transport/SecureSessionMgr.cpp

```diff
@@ -207,6 +207,18 @@ void SecureSessionMgr::ExpireAllPairings(NodeId peerNodeId, FabricIndex fabric)
     }
 }
 
+void SecureSessionMgr::ExpireAllPairingsForFabric(FabricIndex fabric)
+{
+    ChipLogDetail(Inet, "Expiring all connections for fabric %d!!", fabric);
+    PeerConnectionState * state = mPeerConnections.FindPeerConnectionStateByFabric(fabric, nullptr);
+    while (state != nullptr)
+    {
+        mPeerConnections.MarkConnectionExpired(
+            state, [this](const Transport::PeerConnectionState & state1) { HandleConnectionExpired(state1); });
+        state = mPeerConnections.FindPeerConnectionStateByFabric(fabric, nullptr);
```

Comment:
This is worth a comment explaining why the second arg to `FindPeerConnectionStateByFabric` is not `state`.  Presumably the `MarkConnectionExpired` call removed `state` from the initialized list in `mPeerConnections` or something like that?